src/jit/hwintrinsicxarch.cpp

```diff
@@ -403,6 +397,7 @@ bool HWIntrinsicInfo::isFullyImplementedIsa(InstructionSet isa)
         case InstructionSet_FMA:
         case InstructionSet_LZCNT:
         case InstructionSet_POPCNT:
+        case InstructionSet_PCLMULQDQ:
```

Comment:
nit: These look to be alphabetically ordered right now, `PCLMULQDQ` should come before `POPCNT`

Reply:
Okay, will do.